lib/elixir/lib/process.ex

```diff
@@ -784,6 +874,71 @@ defmodule Process do
   @spec hibernate(module, atom, list) :: no_return
   defdelegate hibernate(mod, fun_name, args), to: :erlang
 
+  @type alias_opt :: :explicit_unalias | :reply
+
+  @typedoc """
+  An alias returned by `alias/0` or `alias/1`.
+
+  See [the module documentation](#module-aliases) for more information about aliases.
+  """
+  @type alias :: reference
+
+  @doc """
+  Creates a *process alias*.
```

Comment:
We don't use italics in docs in such scenarios. So removing for consistency. :)

```suggestion
  Creates a process alias.
```